app/javascript/components/journey/types.ts

```diff
@@ -30,6 +30,8 @@ export class TrackProgress {
   progressChart: TrackProgressChart
 
   get completion(): number {
+    if (this.numExercises === 0) return 100
```

Comment:
The UI will show `0/0`, so I think showing this as `100%` completed might make sense. Let me know if you prefer to list it as `0%`.

Reply:
I think 100% is fine.

Reply:
```suggestion
    if (this.numExercises === 0) return 0
```